src/app/containers/Brand/index.jsx

```diff
@@ -1,22 +1,70 @@
 import React, { useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
+import SkipLink from '@bbc/psammead-brand/skip-link';
+import ScriptLink from '@bbc/psammead-script-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { bool } from 'prop-types';
+import { setPreferredVariantCookie } from '#contexts/UserContext/cookies';
 import { ServiceContext } from '#contexts/ServiceContext';
 
+const setCookie = (service, scriptLinkVariant) => {
```

Comment:
Do we need this `setCookie` function, or could you just used `setPreferredVariantCookie` directly?

Reply:
`setPreferredVariantCookie` can be used directly. Switched to using it directly